site/en/docs/web-platform/navigation-api/index.md

```diff
@@ -96,116 +92,96 @@ links.forEach(link => link.addEventListener('click', updatePage));
-If a clicked `<a href>` fails to load and rolls back, the URL won't appear in the user's session history (for Forward or Back navigation).
-
-{% Aside %}
-
-Although calling `rollback()` on error is useful in some situations, you should avoid it in the general case.
-You're more likely to show an error message, perhaps so the user can try a reload to get their content.
-
-{% endAside %}
-
 ### Abort Signals
 
 Since you're able to do asynchronous work while preparing a new page, it's possible that the transition your code is handling (to load a specific URL or state) might get preempted, or considered out-of-date.
-This might happen because the `rollback()` method was invoked, the user just clicked on another link, or your code itself throws an error.
+This might happen because the user just clicked on another link, or your code itself throws an error.
 
-To deal with any of these possibilities, the event passed to the "navigate" handler contains a property called `signal`, which is an instance of `AbortSignal`.
+To deal with any of these possibilities, the event passed to the "navigate" listener contains a `signal` property, which is an `AbortSignal`.
 For more information see [Abortable fetch][abortable-fetch].
 The short version is it basically provides an object that fires an event when you should stop your work.
 Notably, you can pass an `AbortSignal` to any calls you make to `fetch()`, which will cancel in-flight network requests if the navigation is preempted.
 This will both save the user's bandwidth, and reject the `Promise` returned by `fetch()`, preventing any following code from e.g., updating the DOM to show a now invalid page navigation.
 
-For a concrete example, you might set up loading a page of cat memes with a `fetch()` call in your handler.
+For a concrete example, you might set up loading a page of cat memes with a `fetch()` call in your listener.
 By passing the `signal` to it, the fetch will be cancelled if the user decides to instead load a different page on your site before the `fetch` completes.
 Take a look:
 
 ```js
-appHistory.addEventListener('navigate', event => {
-  if (isCatsUrl(event.destination.url)) {
+navigation.addEventListener('navigate', navigateEvent => {
+  if (isCatsUrl(navigateEvent.destination.url)) {
     const method = async () => {
-      const request = await fetch('/cat-memes.json', {signal: event.signal});
+      const request = await fetch('/cat-memes.json', {
+        signal: navigateEvent.signal,
+      });
       const json = await request.json();
       // TODO: do something with cat memes json
     };
-    event.transitionWhile(method());
+    navigateEvent.transitionWhile(method());
   } else {
     // load some other page
   }
 });
 ```
 
-## App History Entries
+## Navigation Entries
 
-The `AppHistory` interface provides the current entry through its `current` property.
+`navigation.current` provides access to the current entry.
```

Comment:
current -> currentEntry. Maybe do a Ctrl+F to make sure you caught them all...

Reply:
Good catch